tools/pipelines/templates/1ES/build-npm-package.yml

```diff
@@ -426,11 +429,12 @@ extends:
                   # At this point we want to publish the artifact with the bundle size analysis,
                   # but as part of 1ES migration that's now part of templateContext.outputs below. TODO: Delete/Update this comment on successful inline test
                   - task: 1ES.PublishPipelineArtifact@1
-                      displayName: Publish Artifacts - bundle-analysis
-                      condition: and( succeeded(), ne(variables['Build.Reason'], 'PullRequest'), eq(${{ parameters.taskPublishBundleSizeArtifacts }}, true) )
-                      targetPath: ${{ parameters.buildDirectory }}/artifacts/bundleAnalysis
-                      artifactname: bundleAnalysis
-                      publishLocation: Container
+                  inputs:
```

Comment:
Indentation here is funky. Also, displayName and condition should be at the same level as inputs, IIRC; the rest are children of inputs.